pkg/import/importer.go

```diff
@@ -296,6 +301,16 @@ func (i *Importer) constructClusterBuilder(kpConfig config.KpConfig, builder Clu
 		return nil, err
 	}
 
+	canonicalRepo, err := kpConfig.CanonicalRepository()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get canonical repository")
+	}
+
+	serviceAccount, err := kpConfig.ServiceAccount()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get canonical repository")
```

Comment:
should this say failed to get service account?

Reply:
good catch